src/index.js

```diff
@@ -1,39 +1,25 @@
 import { render } from "react-dom";
 import React from "react";
-import { createStore, applyMiddleware } from "redux";
 import { Provider } from "react-redux";
-import createSagaMiddleware from "redux-saga";
-import { routerMiddleware } from "connected-react-router";
-import { createBrowserHistory } from "history";
 import AppRouter from "./routers/AppRouter";
 import "./utils/globals";
 import "./styles/styles.less";
-import rootSaga from "./sagas/index";
 import { hentMoter } from "./data/moter/moter_actions";
 import { hentVeileder } from "./data/veiledere/veileder_actions";
-import { createRootReducer } from "./data/rootState";
-
-const history = createBrowserHistory();
-
-const rootReducer = createRootReducer(history);
-
-const sagaMiddleware = createSagaMiddleware();
-
-const store = createStore(
-  rootReducer,
-  applyMiddleware(routerMiddleware(history), sagaMiddleware)
```

Comment:
Trengs ikke routerMiddleware lenger?

Reply:
Trengs kun dersom man skal bruke connected react router, men jeg ser ikke helt behovet for det biblioteket? Tror forskjellen mellom BrowserRouter kun er at man får lagret History i redux, til bruk i debugging - men har aldri hatt noe nytte av det :P

Reply:
Den er grei 👍